src/pages/help-centre/_article-component.tsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { convertToHash } from './_utility'
+import { Header, Text } from 'components/elements'
+import { LocalizedLink, Localize } from 'components/localization'
+import device from 'themes/device'
+
+type IItems = {
+    translate_text: string
+    is_expanded: boolean
+}
+
+type IProps = {
+    props: IItems
+    translate_text: string
+    is_expanded: boolean
+    is_eu_country?: boolean
+}
```

Comment:
Please change `IProps` to something more meaningful. We also don't usually prefix types with `I`.

Reply:
Got it. Change name on most relevant here - CategoryType